spec/spec.md

```diff
@@ -499,6 +490,31 @@ Below is a non-normative example of a decoded VC in a JSON format, signed as a J
 ```
 :::
 
+#### [[def:Authentication Response Validation]]
+
+The following checks MUST be made by the Verifier upon receiving the Authorization Response:
+
+##### ID Token Validation
+
+To validate the ID Token received, the Verifier MUST do the following:
+
+1. The Verifier MUST validate ID Token according to the rules defined in section 12.1 of [[ref: SIOPv2 ID1]].
+2. The ID Token MUST be signed by the End-User's DID:
+3. `iss` claim MUST be `https://self-issued.me/v2/openid-vc`.
+4. The signature on the ID Token MUST be validated. Validation is performed against the key obtained from a DID Document. DID Document MUST be obtained by resolving a Decentralized Identifier included in the `sub` claim using DID Resolution.
+5. `sub` claim MUST equal the `id` property in the DID Document.
+6. The `_vp_token` claim MUST be present and contain a presentation_submission with a valid descriptor map.
+
+##### VP Token Validation
```

Comment:
I think these steps should be more granular. the reason why OpenID4VP spec does not give these detailed steps is because these are credential format specific, so it is great that we now have this profile to clarify.

Reply:
Can be more granular if we come up with a list.

Reply:
Will be addressed via #73